connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -400,6 +410,52 @@ public WorkerConfig(ConfigDef definition, Map<String, String> props) {
         logInternalConverterDeprecationWarnings(props);
     }
 
+    public static void validateHttpResponseHeaderConfig(String config) {
```

Comment:
Why not implement these as a `ConfigDef.Validator` implementation, similar to the existing `AdminListenersValidator` below?